test/controllers/users.test.js

```diff
@@ -97,3 +113,72 @@ describe(`SignUp endpoint POST ${USERS_PATH}`, () => {
     );
   });
 });
+
+describe(`SignIn endpoint POST ${SIGN_IN_PATH}`, () => {
+  beforeEach(async () => {
+    await request(app)
```

Comment:
In this suite you are testing the singing flow, for that reason you should avoid to use other endpoints, to do that you can use factory-girl to create users directly in data base.

Reply:
The suite consume only the users/sessions endpoint. solved. In the next pull request, I optimize the use of factory-girl.